companion/ViewController.swift

```diff
@@ -7,19 +7,55 @@
 //
 
 import UIKit
+import iBeaconManager
```

Comment:
Just a suggestion, this model should be contained into a `BeaconManger` model. Thus, using dependency injection of your model vs the iBeaconManger framework

Reply:
After reading this comment, let me know so I can resolve this conversation.